packages/components/popover/popover.component.ts

```diff
@@ -260,6 +258,11 @@ export class KbqPopoverTrigger extends KbqPopUpTrigger<KbqPopoverComponent> impl
     set trigger(value: string) {
         if (value) {
             this._trigger = value;
+
+            if (this.trigger === PopUpTriggers.Hover) {
```

Comment:
Может сделать проверку на подстроку?
```suggestion
            if (this.trigger.includes(PopUpTriggers.Hover)) {
```